readthedocs/proxito/middleware.py

```diff
@@ -113,4 +117,29 @@ def process_request(self, request):  # noqa
         # Otherwise set the slug on the request
         request.host_project_slug = request.slug = ret
 
+        project = Project.objects.get(slug=request.host_project_slug)
+
+        # This is hacky because Django wants a module for the URLConf,
+        # instead of also accepting string
+        if project.has_feature(Feature.PROJECT_URL_ROUTES) and project.urlconf:
+            class fakeurlconf:
+                urlpatterns = [
+                    re_path(project.real_urlconf, ServeDocs.as_view()),
+                    re_path('^' + project.real_urlconf, ServeDocs.as_view()),
+                    re_path('^/' + project.real_urlconf, ServeDocs.as_view()),
```

Comment:
Just for debugging

Reply:
I had to bring in the 404, 500, and all of the proxito urls here as well.

Reply:
@agjohnson I didn't need the 404/500 yet, but I might be missing something.

Reply:
I've gone ahead and added them.